tests/frontend/test_tool_calling_sglang.py

```
@@ -68,26 +70,108 @@ def _prepare_log_dir(request, suffix: str) -> str:
+        "Post-teardown straggler sweep: sending SIGKILL to %d process(es): %s",
+        len(procs),
+        [(p.pid, (p.info.get("name") or "")[:40]) for p in procs],
+    )
+    for proc in procs:
+        try:
+            proc.send_signal(signal.SIGKILL)
+        except (psutil.NoSuchProcess, psutil.AccessDenied):
+            continue
+
+    # Wait for processes to actually exit before the next topology boots.
+    gone, alive = psutil.wait_procs(procs, timeout=timeout)
```

Ruff (0.15.10)

[warning] Unpacked variable `gone` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)